### PR TITLE
behnaz/btn-img-update

### PR DESCRIPTION
**Pull request by behnaz-fs**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/landing/weekend-trading/components/_hero.js [hunk 1]
```diff
@@ -123,13 +118,7 @@ const HeroComponent = ({ title, content, background_data }) => {
...
                         <Header type="subtitle-1">{content}</Header>
                     </HeroContent>
-                    <TryButton
-                        target="_blank"
-                        rel="noopener noreferrer nofollow"
-                        type="submit"
-                        secondary="true"
-                        to={'/signup/'}
-                    >
+                    <TryButton secondary="true" to={'/signup/'}>
```
Comment (sean-binary) [inline src/pages/landing/weekend-trading/components/_hero.js:121]:
```suggestion
                    <TryButton secondary="true" to="/signup/">
```

Comment (sean-binary) [reply src/pages/landing/weekend-trading/components/_hero.js:121]: The usual convention is to write the path in double quotes.

File: src/pages/landing/weekend-trading/components/_title-btn.js [hunk 2]
```diff
@@ -44,14 +44,7 @@ const Titlebtn = ({ btnlabel, text }) => {
...
                 </StyledText>
-                <TryButton
-                    m="4.2rem 0 40px"
-                    target="_blank"
-                    rel="noopener noreferrer nofollow"
-                    type="submit"
-                    secondary="true"
-                    to={'/signup/'}
-                >
+                <TryButton m="4.2rem 0 40px" secondary="true" to={'/signup/'}>
```
Comment (sean-binary) [inline src/pages/landing/weekend-trading/components/_title-btn.js:47]: Same here.

File: src/pages/landing/weekend-trading/components/_hero.js (hunk 1 above)
Comment (behnaz-fs) [reply src/pages/landing/weekend-trading/components/_hero.js]: done

File: src/pages/landing/weekend-trading/components/_title-btn.js (hunk 2 above)
Comment (behnaz-fs) [reply src/pages/landing/weekend-trading/components/_title-btn.js]: done

**Review by sean-binary — APPROVED** [review]

**Comment by behnaz-fs** [comment]
no need to this pr, fix done by another pr

**Pull request closed without merging**